Main

```diff
@@ -0,0 +1,26 @@
+/// Program asks for 2 diffrent animals (if the animals are not an option the program will ask to choose another)
```

Comment:
Put these comments in the `main` function so that you can slowly implement them.